DQM/SiStripMonitorHardware/src/CMHistograms.cc

```diff
@@ -147,10 +147,10 @@ void CMHistograms::bookTopLevelHistograms(DQMStore::IBooker & ibooker)
     
   //book map after, as it creates a new folder...
   if (tkMapConfig_.enabled){
-    tkmapCM_[0] = new TkHistoMap("SiStrip/TkHisto","TkHMap_MeanCMAPV",0.,500);
-    tkmapCM_[1] = new TkHistoMap("SiStrip/TkHisto","TkHMap_RmsCMAPV",0.,500);
```

Comment:
things like this should be inspected by the developers/responsibles.
This looks like an example where it may be better to leave the warning of int->bool conversion rather than making the compiler quiet to "hide" a possible bug